plugins/azure/automationAccounts/automationAcctExpiredWebhooks.js

```diff
@@ -0,0 +1,73 @@
+const async = require('async');
+const helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'Automation Account Expired Webhooks',
+    category: 'Automation',
+    domain: 'Management and Governance',
+    severity: 'Medium',
+    description: 'Ensure that Azure Automation webhooks are deleted after they have expired',
```

Comment:
I think its better if we rephrase it as webhooks have expiration date configured